python_ta/transforms/ExprWrapper.py

```diff
@@ -189,6 +192,97 @@ def parse_container_op(
         """Convert an astroid List, Set, Tuple node to a list of z3 expressions."""
         return [self.reduce(element) for element in node.elts]
 
+    def apply_in_op(
+        self, left: z3.ExprRef, right: Union[z3.ExprRef, List[z3.ExprRef], str], negate=False
+    ) -> z3.ExprRef:
+        """
+        Apply `in` or `not in` operator on a list or string and return the
+        resulting z3 expression. Raise Z3ParseException if the operands
+        do not support `in` operator
+        """
+        if isinstance(right, list):  # container type (list/set/tuple)
+            return (
+                z3.And(*[left != element for element in right])
+                if negate
+                else z3.Or(*[left == element for element in right])
+            )
+        elif isinstance(right, (str, z3.SeqRef)):  # string literal or variable
+            return z3.Not(z3.Contains(right, left)) if negate else z3.Contains(right, left)
+        else:
+            op = "not in" if negate else "in"
+            raise Z3ParseException(
+                f"Unhandled binary operation {op} with operator types {left} and {right}."
+            )
+
+    def _parse_number_literal(self, node) -> Optional:
```

Comment:
1. `node` is missing a type annotation
2. the return type is `Optional[Union[int, float]]`